XenModel/Messages.resx

```diff
@@ -11627,6 +11627,9 @@ Check your settings and try again.</value>
   <data name="UPDATES_WIZARD_CANNOT_MIGRATE_VM_LICENSE_REASON" xml:space="preserve">
     <value>Cannot migrate VM '{0}' due to license restrictions.</value>
   </data>
+  <data name="UPDATES_WIZARD_CANNOT_MIGRATE_VM_SUSPEND_REASON" xml:space="preserve">
+    <value>Cannot migrate VM '{0}', because it lacks the ability to suspend. Please ensure it has up-to-date drivers installed.</value>

```

Comment:
Although intransitive usage of "suspend" does exist, I think transitive usage is more correct in this context. I would write simply "Cannot migrate VM '{0}', because it cannot be suspended. Please etc."